src/Farmer/Arm/App.fs

```diff
@@ -175,6 +175,7 @@ type ContainerApp =
 
 
 
+
```

Comment:
remove?

Reply:
i think the fantomas thing did this (not sure why)

Reply:
will fix.